api/commands/run.ts

```diff
@@ -300,6 +300,12 @@ export class Runner {
       actionResult.status = dataform.ActionResult.ExecutionStatus.SUCCESSFUL;
     }
 
+    if (
+      action.actionDescriptor &&
+      actionResult.status === dataform.ActionResult.ExecutionStatus.SUCCESSFUL
+    ) {
```

Comment:
do you think we should put this behind a temporary `RunConfig` flag (defaulting to running this code), so that we can turn it off if necessary?